.github/workflows/Documentation.yml

```
@@ -15,7 +15,7 @@ jobs:
       - name: Set up Julia
         uses: julia-actions/setup-julia@v1
         with:
-          version: '1.6'
+          version: '1.7'
```

Maybe
```suggestion
          version: '1'
```
so always the stable version is tested and we don't have to adjust CI for Julia 1.8 etc. manually?